third_party/js.bzl

```diff
@@ -430,3 +430,13 @@ def tensorboard_js_workspace():
       path = "/test-fixture",
       exclude = ["test/**"],
   )
+
+  filegroup_external(
+      name = "org_tensorflow_tfjs",
+      licenses = ["notice"],  # Apache 2.0
+      sha256_urls = {
+          "fccd26db2da462ec48e2d90fbdff1ee9a9d740f2c7efbd9789ba46eb98ecd1ae": [
+              "https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@0.11.5/dist/tf.min.js",
```

Comment:
I believe Justine usually recommends having a mirror.bazel.build link for every one of these.

Also, is it possible to get this from a slightly more canonical location?  E.g. either the GitHub release tarball the NPM registry tarball (https://registry.npmjs.org/@tensorflow/tfjs/-/@tensorflow/tfjs-0.11.5.tgz), and then using web_library_external?

Reply:
Hm, yeah the src links might have to change, fair enough - I can take a look post-merge at some point.